db-tests/src/test/java/TestTriggers.java

```diff
@@ -1067,5 +1068,105 @@ SELECT COUNT(*)
         }
       }
     }
+
+    @Test
+    void shouldCancelOnMissionModelUpdate() throws SQLException {
```

Comment:
Should cancel what? Can you rename this to be more clear? Since all the trigger tests are in this one Java class, it's helpful to have very descriptive names as you've been doing in previous work. This applies to the tests below as well.

Reply:
Done